packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -107,6 +108,7 @@ const DepsDisplayName = {
   [DepsType.Dotnet]: ".NET Core SDK",
   [DepsType.Ngrok]: "ngrok",
   [DepsType.FuncCoreTools]: "Azure Functions Core Tools",
+  [DepsType.VxTestApp]: "Video Filter Test App",
```

Comment:
Let's use `Video Extensibility Test App` to keep consistent.
![image](https://user-images.githubusercontent.com/15644078/201268954-7dce4e06-6f59-4568-be34-f2c07507591d.png)

Reply:
Updated